### PR TITLE
Solved a bug where mask_fmap and the dilated mask where switched

### PR DESCRIPTION
**Pull request by Aurelienpujolm**
## Checklist

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied the relevant labels to this PR
- [x] I've added relevant tests for my contribution
- [x] I've updated the documentation and/or added correct docstrings
- [x] I've assigned a reviewer

## Description
I've just changed the index to get the mask_fmap and the dilated_mask in the sequencer, because I inverted them in the last PR

### CODE REVIEW

File: shimmingtoolbox/shim/sequencer.py [hunk 1]
```diff
@@ -327,8 +327,8 @@ def get_resampled_masks(self):
                 for i in range(n_shims))
 
             # We need to transpose the mask to have the good dimensions
-            resampled_mask = np.array([results_mask[it][0].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
-            masks_fmap = np.array([results_mask[it][1].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
+            resampled_mask = np.array([results_mask[it][1].get_fdata() for it in range(n_shims)]).transpose(1, 2, 3, 0)
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]: Can you give more descriptive names to the variables? What is resampled_mask? How is it different from masks_fmap?

Comment (Aurelienpujolm) [reply shimmingtoolbox/shim/sequencer.py]: We can probably go with resampled_mask_dilated or something like that ? 
I think that we should probably go with another PR after this one to fixed all of the confused names in the code !

Comment (po09i) [reply shimmingtoolbox/shim/sequencer.py]: I suggest improving the ones you actively interact with in the PR you are working. I would put a dedicated PR for that as low priority but we can keep it in the back of our minds when we do some changes to improve the codebase.

Comment (Aurelienpujolm) [reply shimmingtoolbox/shim/sequencer.py]: So for this PR, is masks_fmap_dilated, and masks_fmap good for you ?

File: shimmingtoolbox/shim/sequencer.py [hunk 2]
```diff
@@ -82,12 +82,12 @@ def __init__(self, slices, mask_dilation_kernel, mask_dilation_kernel_size, reg_
...
         self.optimizer = None
 
-    def optimize(self, resampled_mask):
+    def optimize(self, masks_fmap_dilated):
         """
         Optimization of the currents for each shim group. Wraps :meth:`shimmingtoolbox.shim.sequencer.Sequencer.opt`.
 
         Args:
-            resampled_mask (np.ndarray): 3D anat mask used for the optimizer to shim in the region of interest.
+            masks_fmap_dilated (np.ndarray): 3D anat mask used for the optimizer to shim in the region of interest.
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]: Is this right, the name says that it is the fmap mask and the description says that it is the anat

**Review by po09i — CHANGES_REQUESTED** [review]

File: shimmingtoolbox/shim/sequencer.py (hunk 2 above)
Comment (Aurelienpujolm) [reply shimmingtoolbox/shim/sequencer.py]: good catch, I will fixed this

File: shimmingtoolbox/shim/sequencer.py [hunk 3]
```diff
@@ -82,13 +82,13 @@ def __init__(self, slices, mask_dilation_kernel, mask_dilation_kernel_size, reg_
...
 
-    def optimize(self, resampled_mask):
+    def optimize(self, masks_fmap_dilated):
         """
         Optimization of the currents for each shim group. Wraps :meth:`shimmingtoolbox.shim.sequencer.Sequencer.opt`.
 
         Args:
-            resampled_mask (np.ndarray): 3D anat mask used for the optimizer to shim in the region of interest.
-                                             (only consider voxels with non-zero values)
+            masks_fmap_dilated (np.ndarray): 3D fieldmap mask used for the optimizer to shim in the region
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]:
```suggestion
            masks_fmap (np.ndarray): 3D fieldmap mask used for the optimizer to shim in the region
```

File: shimmingtoolbox/shim/sequencer.py [hunk 4]
```diff
@@ -82,13 +82,13 @@ def __init__(self, slices, mask_dilation_kernel, mask_dilation_kernel_size, reg_
         self.path_output = path_output
         self.optimizer = None
 
-    def optimize(self, resampled_mask):
+    def optimize(self, masks_fmap_dilated):
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]:
```suggestion
    def optimize(self, masks_fmap):
```

File: shimmingtoolbox/shim/sequencer.py [hunk 5]
```diff
@@ -97,12 +97,12 @@ def optimize(self, resampled_mask):
         coefs = []
         for i in range(n_shims):
             # If there is nothing to shim in this shim group
-            if np.all(resampled_mask[..., i] == 0):
+            if np.all(masks_fmap_dilated[..., i] == 0):
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]:
```suggestion
            if np.all(masks_fmap[..., i] == 0):
```

File: shimmingtoolbox/shim/sequencer.py [hunk 6]
```diff
@@ -97,12 +97,12 @@ def optimize(self, resampled_mask):
...
                 coefs.append(np.zeros(self.optimizer.merged_coils.shape[-1]))
 
             # Otherwise optimize
             else:
-                coefs.append(self.optimizer.optimize(resampled_mask[..., i]))
+                coefs.append(self.optimizer.optimize(masks_fmap_dilated[..., i]))
```
Comment (po09i) [inline shimmingtoolbox/shim/sequencer.py]:
```suggestion
                coefs.append(self.optimizer.optimize(masks_fmap[..., i]))
```

**Review by po09i — CHANGES_REQUESTED** [review]

**Review by po09i — APPROVED** [review]

**Pull request merged**